src/server.c

```diff
@@ -4729,6 +4782,17 @@ sds genRedisInfoString(const char *section) {
             server.blocked_clients,
             server.tracking_clients,
             (unsigned long long) raxSize(server.clients_timeout_table));
+        for (j = 0; j < CLIENT_MEM_USAGE_BUCKETS; j++) {
+            info = sdscatprintf(info, "clients_mem_bucket%02d: tot-mem: %zu, clients: %lu\r\n", j,
+                                server.client_mem_usage_buckets[j].mem_usage_sum,
+                                server.client_mem_usage_buckets[j].clients->len);
+            listIter *it = listGetIterator(server.client_mem_usage_buckets[j].clients, AL_START_HEAD);
+            for (listNode *ln = listNext(it); ln; ln = listNext(it)) {
+                client *c = (client*)ln->value;
+                info = sdscatprintf(info, "client_mem: %zu\r\n", c->client_last_memory_usage);
+            }
```

Comment:
we can't afford a loop on all clients in INFO.
but this can be either moved in some way to CLIENT LIST, or a new CLIENT subcommand.

Reply:
This was a debugging tool I added, note that `mem_usage_sum` is calculated per bucket just for debugging. The first question is do we even need this (after we're done debugging). If not I'll just delete the whole thing. If we do want it then I'll think where best to put it. WDYT?

Reply:
i suppose we don't need it.
if we do, then a DEBUG sub-commend is the right place.
the other alternatives of a new CLIENT sub-command, or an additional field in CLIENT LIST, are not desired IMHO since this is detail about internal implementation that may change some day.

Reply:
moved to `debug client-eviction`